addon/components/new-bourbon-select-field.js

```diff
@@ -102,7 +102,15 @@ export default Component.extend({
         'value',
         this.get('value')
       );
-      return this.get('prompt') ? this.get('internalContent').indexOf(valueHolder) : 0;
+
+      let index = this.get('internalContent').indexOf(valueHolder);
+
+      if (index !== -1) {
+        return index;
+      } else if (index === -1 && !this.get('prompt')) {
```

Comment:
I don't think we need `index === -1` here, since if it were equal to `-1`, the first `if` condition above would have caught it, right?

Reply:
yeah...i think you are right about this.  i will remove it.